wicket-extensions/src/test/java/org/apache/wicket/extensions/markup/html/AjaxLazyLoadPanelTesterTest.java

```diff
@@ -38,12 +38,12 @@
 	@Test
 	public void test()
 	{
-		AjaxLazyLoadPanel panel = new AjaxLazyLoadPanel("panel")
+		AjaxLazyLoadPanel<Component> panel = new AjaxLazyLoadPanel<Component>("panel")
 		{
 			private static final long serialVersionUID = 1L;
 
 			@Override
-			public Component getLazyLoadComponent(final String markupId)
+			protected Component getLazyLoadComponent(String markupId)

```

Comment:
I guess `markupId` can remain `final` here ...

Reply:
Thanks, it should still be public too.